doc/source/howtos/encrypt-traffic.rst

```diff
@@ -236,4 +236,46 @@ assumig there are two nodes in the cluster:
 
       wsrep_provider_options=socket.ssl=yes;socket.ssl_ca=/path/to/upgrade-ca.pem;socket.ssl_cert=/path/to/new-cert.pem;socket.ssl_key=/path/to/new-key.pem
 
+.. _ssl-auto-conf:
+
+SSL Automatic Configuration
+===========================
+
+|PXC| includes the :variable:`pxc-encrypt-cluster-traffic` variable
+that enables automatic configuration of SSL encrytion.
+By default, it is disabled, meaning that you need to configure SSL manually
```

Comment:
Please this stmt

By default, variable is false. Turning it to true will auto-configure use of SSL encryption for SST, IST and internode traffic.